functions/Get-DbaClusterSqlInstance.ps1

```diff
@@ -0,0 +1,84 @@
+function Get-DbaClusterSqlInstance
```

Comment:
Oh, sorry, I overwrote that comment, but it got lost apparently
I wanted this to say:
We could have more functions later that collect the nodes (I wrongly assumed first that's what this function did), or work with SQL Server FCI.
So I think it would be better to make clear this is about WSFC cluster and resources, not nodes.
Get-DbaWsfcSqlResource?